packages/cloudflare/src/index.ts

```diff
@@ -197,7 +139,8 @@ export default function createIntegration(args?: Options): AstroIntegration {
 				});
 			},
 			'astro:server:setup': async ({ server }) => {
-				if (args?.platformProxy?.enabled === true) {
+				const usesPlatformProxy = args?.platformProxy?.enabled ?? true;
+				if (args?.platformProxy && usesPlatformProxy === true) {
```

Comment:
is this code correct?
the `if` condition disables the use of `platformProxy` if it's not specified in the args, which I think it's not what we want?

what about something like this instead?
![Screenshot 2024-06-19 at 23 44 31](https://github.com/withastro/adapters/assets/61631103/3f0a229f-ea1b-4e29-a6c9-28da56979f3c)